lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
+ *
+ * @param {!Node} element
```

Comment:
I'm not familiar with the problem we are attempting to solve here. Can you describe why it is important to be using elements instead of nodes?